docker tagging no space

https://github.com/Project-MONAI/MONAI/actions/runs/6274006320/job/17039074690
```
--------------------
ERROR: failed to solve: failed to register layer: write /usr/local/nvm/.cache/bin/node-v16.20.0-linux-x64/node-v16.20.0-linux-x64.tar.xz: no space left on device
Error: Process completed with exit code 1.
```

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -141,6 +141,7 @@ jobs:
           RELEASE_VERSION: ${{ steps.versioning.outputs.tag }}
         shell: bash
         run: |
+          rm -rf /opt/hostedtoolcache
           # get tag info for versioning
           mv _version.py monai/
           # version checks

--- a/monai/networks/nets/segresnet_ds.py
+++ b/monai/networks/nets/segresnet_ds.py
@@ -127,7 +127,7 @@ class SegResBlock(nn.Module):
 
 class SegResEncoder(nn.Module):
     """
-    SegResEncoder based on the econder structure in `3D MRI brain tumor segmentation using autoencoder regularization
+    SegResEncoder based on the encoder structure in `3D MRI brain tumor segmentation using autoencoder regularization
     <https://arxiv.org/pdf/1810.11654.pdf>`_.
 
     Args:

--- a/monai/transforms/utils.py
+++ b/monai/transforms/utils.py
@@ -187,7 +187,7 @@ def rescale_array(
     Args:
         arr: input array to rescale.
         minv: minimum value of target rescaled array.
-        maxv: maxmum value of target rescaled array.
+        maxv: maximum value of target rescaled array.
         dtype: if not None, convert input array to dtype before computation.
 
     """

